build.sh

```diff
@@ -17,13 +17,18 @@
 ROOT_DIR="${PWD}"
 SCRIPTS_DIR="${ROOT_DIR}/scripts"
 
+DOCKERFILE="Dockerfile"
 AUTOTUNE_DOCKER_REPO="kruize/autotune"
 # Fetch autotune version from the pom.xml file.
 AUTOTUNE_VERSION="$(grep -A 1 "autotune" "${ROOT_DIR}"/pom.xml | grep version | awk -F '>' '{ split($2, a, "<"); print a[1] }')"
 AUTOTUNE_DOCKER_IMAGE=${AUTOTUNE_DOCKER_REPO}:${AUTOTUNE_VERSION}
+DEV_MODE=0
 
 function usage() {
-	echo "Usage: $0 [-v version_string] [-t docker_image_name]"
+	echo "Usage: $0 [-d] [-v version_string] [-t docker_image_name]"
+	echo " -d: build in dev friendly mode"
+	echo " -v: build as specific autotune version"
+	echo " -t: build with specific docker image tag"
```

Comment:
Change docker image tag to docker image name

Reply:
Should be fixed now